services/order.py

```diff
@@ -0,0 +1,39 @@
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, User
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
```

Comment:
The `date` parameter should be a `datetime` object instead of a `str` to ensure proper handling of date and time. Consider using `from datetime import datetime` and parsing the string to a datetime object before using it.